app/src/main/java/net/squanchy/schedule/view/EventsAdapter.java

```diff
@@ -28,7 +28,7 @@
 
     @Override
     public long getItemId(int position) {
-        return events.get(position).id();
+        return Long.parseLong(events.get(position).id());
```

Comment:
❌  There is no contract saying this is a number. We should probably put an additional `int` or `long` `numericId` field in `Event` that we calculate when we convert an API event into a proper event domain model. I am considering which hashing algorithm we could use; [CRC](https://developer.android.com/reference/java/util/zip/CRC32.html) seems a good starting point.